### ISSUE TITLE
Change icon font dynamically

### ISSUE THREAD

**Issue by rafaelcruz-net** [junalmeida/dash-ci#24]
In my work google fonts API is blocked by proxy.
Is it possible to change icon font to fontawesome or other else icon font for dash ci ?

Thxns

**Comment by junalmeida** [comment]
its kinda stupid decision for a company to block google apis, cdns, etc.

**Comment by rafaelcruz-net** [comment]
new pull request for this issue, now dashboard load font from loca
